R/calculate_pga.R

```diff
@@ -21,8 +21,10 @@
 #' @export
 #'
 #' @examples
+#' library(dplyr)
```

Comment:
Very minor comment, but perhaps we would want to prefix the `filter` function with `dplyr` to avoid any potential conflicting function names (we do this consistently throughout the full GAMBLR code-base)? I think this would also avoid having to load `dplyr` at the beginning of the example. Let me know what you think.

Reply:
Sure! I agree. 
I made the change. Please, let me know if there is anything else to change.